src/main/java/org/apache/jackrabbit/filevault/maven/packaging/GenerateMetadataMojo.java

```diff
@@ -410,6 +421,11 @@ public void execute() throws MojoExecutionException, MojoFailureException {
         final File vaultDir = getVaultDir();
         vaultDir.mkdirs();
 
+        // JCRVLT-331 share work directory to expose vault metadata between process-classes and package phases for
+        // multi-module builds.
+        getArtifactWorkDirectoryLookup(getPluginContext())
+                .put(getModuleArtifactKey(project.getArtifact()), workDirectory);
```

Comment:
I think using Artifact.getId() as key is simpler here

Reply:
See my explanation below. The artifact.getId() value here would contain `:content-package:` instead of `:zip:`, but the module lookup needle Artifact.getId() would contain `:zip:`, and so wouldn't match.